app/users/testing.go

```diff
@@ -0,0 +1,23 @@
+package users
+
+import (
+	"errors"
+
+	"github.com/lbryio/lbrytv/models"
+)
+
+// TestUserRetriever is a helper allowing to test API endpoints that require authentication
+// without actually creating DB records.
+type TestUserRetriever struct {
+	AccountID string
+	Token     string
+}
+
+// Retrieve returns AccountID set during TestUserRetriever creation,
```

Comment:
You have the function comments here both a lot of other places there are no comments on exported functions, constants and vars.